src/layouts/Base.astro

```diff
@@ -69,10 +73,7 @@ const jsonLDSchema = JSON.stringify(
 			href="/fonts/outfit-v6-latin-700.woff2"
 			crossorigin
 		/>
-		<script
-			type="application/ld+json"
-			set:html={JSON.stringify(jsonLDSchema, null, 0)}
-		></script>
```

Comment:
🔍 By stringifying this data again here, I was causing the JSON to be injected as a string rather than as a JSON object. To see what I mean, inspect the source code of the live site and check out the contents of the json+ld script tag.

Reply:
Sure enough!